cuba.yml

```diff
@@ -545,3 +545,23 @@ CUBA_KEYS:
     definition: Classification of a fluid flow type
     shape: [1]
     type: integer
+
+  VECTOR:
+    definition: A vector
+    shape: [1]
+    type: double
+
+  POINT:
+    definition: A point in a 3D space system
+    shape: [3]
+    type: double
+
+  FACE:
+    definition: A face
+    shape: [1]
+    type: double
```

Comment:
Face is defined in simphony.cuds.mesh and should have more info than just a numeric (double).
Could BOX use mesh or lattice internally and not have to define FACE here?